.github/workflows/checkimage.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
           base=$(grep -Po '(?<=FROM )([^\s]*)(?= AS build)' Dockerfile)
           skopeo inspect "docker://$base" | jq .Digest --raw-output > .baseimagedigest
           docker run --rm -u 0 quay.io/cloudservices/compliance-backend:latest sh -c \
-            'microdnf update -y $(cat /opt/installedpackages) > /dev/null; rpm -q $(cat /opt/installedpackages) | sort | sha256sum | cut -d " " -f 1' \
+            'rpm -qa | grep -v "gpg-pubkey" > /opt/installedpackages; microdnf update -y $(cat /opt/installedpackages) > /dev/null; rpm -q $(cat /opt/installedpackages) | sort | sha256sum | cut -d " " -f 1' \
```

Comment:
Please review these container build arguments that include packages that are being installed, that also might be used for tracking changes: https://github.com/RedHatInsights/compliance-backend/blob/e90e9177f581687e0b0801b5876e5f23c48682ff/Dockerfile#L1-L2

Reply:
ping @romanblanco